3drepo/3drepo.io
# ISSUE #3160 - Change styling of 'Add new Federation/Container' at the bottom of lists

This fixes #3160 

#### Description
- Change the styling to make the buttons less bright and distracting


#### Test cases
- hover over button.
- click and hold button to see active state is the same as hover
